docs/navigation.md

```diff
@@ -3,72 +3,26 @@ id: navigation
 
-This guide covers the various navigation components available in React Native. If you are just getting started with navigation, you will probably want to use [React Navigation](navigation.md#react-navigation). React Navigation provides an easy to use navigation solution, with the ability to present common stack navigation and tabbed navigation patterns on both iOS and Android.
+|                                                                                                           | Uses native API | Has one React root | Runs in Expo | Runs on the Web |
+| --------------------------------------------------------------------------------------------------------- | --------------- | ------------------ | ------------ | --------------- |
+| [React Navigation](https://reactnavigation.org/docs/en/getting-started.html)                              | No              | Yes                | Yes          | No              |
+| [React Native Navigation](https://wix.github.io/react-native-navigation/)                                 | Yes             | No                 | No           | No              |
+| [The Navigation router](https://grahammendick.github.io/navigation/documentation/native/hello-world.html) | Yes             | Yes                | No           | Yes             |
 
-If you'd like to achieve a native look and feel on both iOS and Android, or you're integrating React Native into an app that already manages navigation natively, the following library provides native navigation on both platforms: [react-native-navigation](https://github.com/wix/react-native-navigation).
+## Uses native API
 
-## React Navigation
+React Native uses the same fundamental UI building blocks as regular apps. Both iOS and Android have fundamental navigation building blocks Android has `Activities` or `Fragments` and iOS has `UIViewControllers`. If your navigation library doesn't use them then you can end up with an app that doesn't have a native look and feel. You'll also miss out on iOS navigation bar features like large titles and search bars.
 
-The community solution to navigation is a standalone library that allows developers to set up the screens of an app with just a few lines of code.
+## Has one React root
 
-The first step is to install in your project:
+If your navigation library renders all screens under a single React root then they share the React `Context`. That means you can use any state management library, or none, to pass data between them. If screens don't share the React `Context` then you have to hold the shared data in a global store outside of the React tree. That rules out state management solutions like Relay and Apollo and only rules in Redux.
```

Comment:
This section should explain the benefits of having separate roots to remain unbiased

Reply:
Its untrue that you can't use Apollo if you have multiple roots like in react-native-navigation. Of course you can share a context with multiple roots.